common/analysis/processingStep.js

```diff
@@ -65,6 +76,37 @@ export const dissocCalculation = (calculation) =>
     )
   )
 
+export const assocVariablesPreviousStep = (variablesPreviousStep) =>
+  ObjectUtils.setProp(keysProps.variablesPreviousStep, variablesPreviousStep)
+
+export const initializeVariablesPreviousStep = ({ previousStep }) => (step) => {
```

Comment:
why this? don't you need them just in the ui?

Reply:
it has been moved to ChainController. Anyway, you are right, in the db we can keep only the "included" variables.